src/main.js

```diff
@@ -3,6 +3,23 @@ var randomPosterButton = document.querySelector('.show-random');
 var mainPosterImage = document.querySelector('.poster-img');
 var posterTitle = document.querySelector('.poster-title');
 var posterQuote = document.querySelector('.poster-quote');
+
```

Comment:
Thank you for sectioning out the variable lists by type! Much easier to read.

Reply:
Great for programmer empathy